### PR TITLE
VM-84

### PR DESCRIPTION
**Pull request by auroraxcheng**

### CODE REVIEW

**Comment by liraymond04** [comment]
image
I'm getting some errors when I enter the /dispatcher page

image
And also some warnings

Can you fix these before I approve them?

**Comment by auroraxcheng** [comment]
Yes I’ll work on it 
Im not sure why im getting the unhandled runtime error though, this is what i’ms seeing 
￼

File: apps/app/src/components/UserSettingsDashboard/UserDispatcher.tsx [hunk 1]
```diff
@@ -1,7 +1,75 @@
...
+import { signMessage } from '@wagmi/core'
+import React, { useState } from 'react'
+
+import lensClient from '@/lib/lens-protocol/lensClient'
+import { useAppPersistStore } from '@/store/app'
 
 const UserDispatcher: React.FC = () => {
-  return <p>Dispatcher</p>
+  const { currentUser } = useAppPersistStore()
+  const [isEnabled, setIsEnabled] = useState(currentUser!.dispatcher !== null)
```
Comment (liraymond04) [inline apps/app/src/components/UserSettingsDashboard/UserDispatcher.tsx:10]: Changing `currentUser!.dispatcher` to `currentUser?.dispatcher` seems to fix the runtime error

Comment (auroraxcheng) [reply apps/app/src/components/UserSettingsDashboard/UserDispatcher.tsx:10]: changed to currentUser? but now im getting errors for profileId and address: Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.

**Review by liraymond04 — APPROVED** [review]

**Pull request merged**